### ISSUE TITLE
Discussion board grading

### ISSUE THREAD

**Issue by JasonSills** [DS4PS/cpp-524-spr-2020#4]
Hi,

I noticed in the syllabus that discussion participation isn't graded, but in the yellowdig post there is a completion calculator. For example, after I posted I was shown that I was 10% complete with the assignment. Is this calculator used for grading or just an aspect of yellowdig we should ignore?

Thank you!

**Comment by lecy** [comment]
Correct, please ignore the points on Yellowdig. They are set-up when the YellowDig board is created through a course. They will not play into your final grade.
